examples/app_with_observability/app_async.py

```diff
@@ -7,7 +7,11 @@
     "Give me 10 names for a baby from this country {country} with gender {gender}!!!!"
 )
 
-ag.init()
+ag.init(
+    app_id="xxxxxxxx",
+    host="https://cloud.agenta.ai",
+    api_key="xxxxx.xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
+)
 tracing = ag.llm_tracing()
 ag.config.default(
     temperature=ag.FloatParam(0.2), prompt_template=ag.TextParam(default_prompt)
```

Comment:
The @ag.entrypoint line later comes with some challenges. The issue is that it breaks the cli argument for anyone using cli arguments